pkg/volume/fc/fc_util.go

```diff
@@ -60,7 +60,7 @@ func (handler *osIOHandler) WriteFile(filename string, data []byte, perm os.File
 
 // given a wwn and lun, find the device and associated devicemapper parent
 func findDisk(wwn, lun string, io ioHandler, deviceUtil volumeutil.DeviceUtil) (string, string) {
-	fcPath := "-fc-0x" + wwn + "-lun-" + lun
+	fcPath := "fc-0x" + wwn + "-lun-" + lun
```

Comment:
This regex is too loose. Unless there are other naming conventions, we only have `/dev/disk/by-path/fc-0x` and `/dev/disk/by-path/pci-xxxx-fc-0x`. The regex should just filter these cases.

Reply:
@rootfs added the regexp for the above cases, can you ptal at the latest changes?